khal/aux.py

```diff
@@ -139,6 +139,20 @@ def generate_random_uid():
     choice = string.ascii_uppercase + string.digits
     return ''.join([random.choice(choice) for _ in range(36)])
 
+def construct_event_list_from_string(ics_as_string):
+    """Construct a list with icalendar.event types with a random UID"""
+    try:
+        events = icalendar.Calendar.from_ical(ics_as_string)
+    except ValueError:
+        logger.fatal("Please specify a ics file")
```

Comment:
I don't agree on the <code>ValueError</code>. The only way a <code>ValueError</code> could occur here is if the <code>ics_as_string</code> is in fact not a ics_as_string.
Printing the error might be useful with the verbose flag, but without I think it is not useful to the user.

Furthermore this is an error that might occur in normal use. Sometimes people select the wrong file and sometimes files are not correct. So throwing an Error is, in my opinion, too verbose.